submissions/adekemiAwomolo/database/query.sql

```diff
@@ -0,0 +1,23 @@
+--Total numbers of Survivals
+SELECT count(*) FROM accidents WHERE passengers_survived = 1; 
+--342
+--Total numbers of passengers who did not survive the accidents
+SELECT count(*) FROM accidents WHERE passengers_survived = 0;
+--549
+--Name and sex of Passengers under the age of 27 that embarked Queenstown and Cherbourg
+SELECT passengers.full_name, passengers.sex FROM passengers LEFT JOIN passengers_details ON passengers.id = passengers_details.passengers_id

```

Comment:
there is no column called `full_name` from your database task. Kindly check properly